docs/customization/model.rst

```diff
@@ -148,44 +167,61 @@ Just like for regular models you can also check the class of translatable models
 
     $ php bin/console debug:container --parameter=sylius.model.shipping_method.class
 
-**1.** The first thing to do is to write your own class which will extend the base ``ShippingMethod`` class.
+**1.** The first thing to do is to add your own interface ``App\Entity\Shipping\ShippingMethodInterface`` which will extend the base ``Sylius\Component\Core\Model\ShippingMethodInterface`` interface.
+
+.. code-block:: php
+
+    <?php
+
+    declare(strict_types=1);
+
+    namespace App\Entity\Shipping;
+
+    use Sylius\Component\Core\Model\ShippingMethodInterface as BaseShippingMethodInterface;
+
+    interface ShippingMethodInterface extends BaseShippingMethodInterface
+    {
+        public function getEstimatedDeliveryTime(): ?string;
```

Comment:
I guess the example is quite unfortunate here:

1. It looks like it shouldn't be a part of a translation - the estimated delivery time shouldn't depend on the locale one is using.
2. Estimated delivery time looks more like `\DateInterval` (for the time interval from shipping to delivery) or `\DateTimeInterface` (if some service generates it on-the-fly for given shipping).

Reply:
I think that we treat this value that note for customer and it may be only simple string.